src/appshell/qml/Preferences/internal/UiColorsSection.qml

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.3
+
+import MuseScore.UiComponents 1.0
+
+Column {
+    id: root
+    spacing: 18
+
+    property int firstColumnWidth: 0
+
+    signal colorChangeRequested(var newColor, var propertyName)
+
+    StyledTextLabel {
+        text: qsTrc("appshell", "UI Colors")
+        font: ui.theme.bodyBoldFont
+    }
+
+
+    GridLayout {
+        columnSpacing: parent.width/8
+        rowSpacing: 20
+        columns: 2
+        Repeater {
+
+            model: [
+                { textRole: "Accent Color:", colorRole: ui.theme.accentColor},
+                { textRole: "Text and Icons:", colorRole: ui.theme.fontPrimaryColor},
+                { textRole: "Disabled Text:", colorRole: "#000000"},
```

Comment:
I don't quite understand why you don't use ui.theme here. Could you please explain?

Reply:
From what i understand, you're asking about the "Disabled Text" property, right?

As of right now, the Disabled Text property isn't tied to anything i.e. changing the colors of that property doesn't really do anything.
From what i've been able to decipher, we don't have a standard way of referencing "Disabled" items. However, Each Item in QML has an `enabled` property and QWidget has an `enabled` member. Perhaps i need to create a new Theme variable (much like the ones in the `ThemeStyleKey` enum in `uitypes.h`, and then connect it across all UI elements to have a color for disabled properties 🤔 .

Reply:
>From what i understand, you're asking about the "Disabled Text" property, right?

Yes, right

>Perhaps i need to create a new Theme variable (much like the ones in the ThemeStyleKey enum in uitypes.h)

It would be great

Reply:
Understood!
As of now, I'm working on the navCtrl borders (nearly done, will post an update within the next few hours). 
This isn't as high of a priority as score-page color inversion is, so i've decided to hold off on fixing this for a little longer, or until i can finish the score inversion feature.